Entity 매핑, 수정, 추가

Joowon-Seo opened the pull request:
## 📖 설명 📖

- 매핑한 Entity  :  Mentor_Schedule_Template, Mentors, Mentor_Social_Infos, Mentor_Abstract_Accounts, Payments, Chat_Room,  Message

<br>

## 🔧 추가 및 수정 🔧️

- Type 수정 : 원시타입 -> 참조타입으로 수정하였습니다.
- Entity 생성 : Payment, SkillStack, Review Entity를 생성하였습니다.
- Repository 생성 : Mentee 관련, Mentor 관련, OrderSheet, Payment, Review, SelectedClassTime,
Skillstack
- 변수명 수정 : PK 변수명을 id -> domainId 로 변경하였습니다.

<br>

## ✨ 이건 꼭 봐주세요! ✨

- Entity Id 변수명을 기존 id에서 보다 직관적으로 일부 수정하였습니다.(id -> menteeAccountId)
- 확인하신후 보다 좋은 이름으로 변경하셔도 좋을것 같습니다.

Joowon-Seo committed:
Feat : Entity 연관 관계 매핑

Joowon-Seo committed:
Feat : SkillStack Entity  생성

Joowon-Seo committed:
Refactor : Entity 수정
Mentor 관련 Entity, Message Entity PK 변수명, 타입 수정

Joowon-Seo committed:
Feat : Payment Entity 생성

Oh-Myeongjae reviewed (approved):
깔끔하게 잘정리하신것 같습니다. 고생하셨습니다~~!!

Joowon-Seo committed:
Feat : Review Entity 생성

Joowon-Seo committed:
Feat : Repository 생성
Mentee 관련, Mentor 관련, OrderSheet, Payment, Review, SelectedClassTime,
Skillstack Repository 생성

koeyismyusername reviewed (approved):
감사합니다 주원님! 원시타입보다는 참조타입, id도 더 직관적이게 앞으로 사용해야겠군요! 아 참 그리고 Repository 생성하셨는데 @Repository 어노테이션이 빠진 것 같습니다!

koeyismyusername committed:
Merge branch 'develop' into feature/base

Joowon-Seo commented:
@koeyismyusername 
민혁님 꼼꼼하신 리뷰 감사합니다!
```@ComponentScan``` 때문에 정상작동을 하다보니 제가 빼먹은 것 같습니다!
Repository annotation 추가하겠습니다! 😄

hjm12223 reviewed (commented):
수정해주셔서 감사합니다!
